specification/dataprotection/resource-manager/Microsoft.DataProtection/DataProtection/BackupVaultResource.tsp

```diff
@@ -223,6 +258,58 @@ interface BackupVaultOperationResults {
   >;
 }
 
+/**
+ * Deleted backup vault operations (available from version 2025-09-01)
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-interface-requires-decorator" "DeletedBackupVaults interface uses manual routing for location-scoped resources"
+@added(Versions.v2025_09_01)
+interface DeletedBackupVaults {
+  /**
+   * Gets a deleted backup vault
+   */
+  @get
+  @route("/subscriptions/{subscriptionId}/providers/Microsoft.DataProtection/locations/{location}/deletedVaults/{deletedVaultName}")
+  get(
+    ...ApiVersionParameter,
+    ...SubscriptionIdParameter,
+    ...LocationParameter,
+
+    /**
+     * Name of the deleted backup vault
+     */
+    @path deletedVaultName: string,
+  ): ArmResponse<DeletedBackupVaultResource> | ErrorResponse;
+
+  /**
+   * Lists deleted backup vaults by location
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-operation" "Manual routing required for location-scoped resource listing"
+  @list
+  @get
+  @route("/subscriptions/{subscriptionId}/providers/Microsoft.DataProtection/locations/{location}/deletedVaults")
+  listByLocation(
+    ...ApiVersionParameter,
+    ...SubscriptionIdParameter,
+    ...LocationParameter,
+  ): Azure.Core.Page<DeletedBackupVaultResource> | ErrorResponse;
+
+  /**
+   * Undeletes a soft deleted backup vault
+   */
+  @post
+  @route("/subscriptions/{subscriptionId}/providers/Microsoft.DataProtection/locations/{location}/deletedVaults/{deletedVaultName}/undelete")
+  undelete(
```

Comment:
Same comment, this is just a resource action

Reply:
Removed this API